packages/flutter_tools/test/commands.shard/hermetic/assemble_test.dart

```diff
@@ -18,15 +18,13 @@ import '../../src/testbed.dart';
 void main() {
   Testbed testbed;
   MockBuildSystem mockBuildSystem;
-
-  setUpAll(() {
-    Cache.disableLocking();
-  });
+  Cache.disableLocking();
```

Comment:
Why didn't `setUpAll()` work?

Reply:
I couldn't figure this one out. I think it was a combination of issues and this change was unnecessary.